ramls/change-due-date.raml

```diff
@@ -0,0 +1,38 @@
+#%RAML 1.0
+title: Change due date
+version: v0.1
+protocols: [ HTTP, HTTPS ]
+baseUri: http://localhost:9130
+
+documentation:
+  - title: API for changing due date for loans
+    content: <b>Change loan due date API</b>
+
+types:
+  errors: !include raml-util/schemas/errors.schema
+
+traits:
+  validate: !include raml-util/traits/validation.raml
+
+/circulation/loans/{id}/change-due-date:
+  post:
+    is: [validate]
+    body:
+      application/json:
+        type: !include change-due-date-request.json
+    responses:
+      204:
```

Comment:
I think it should be 201 instead with the updated Loan in response.

What do you think?

Reply:
I believe 201 refers to resource creation, what resource would be created in response to this API request?

Reply:
If we're going to respond with updated loan then it should be 201 I think, because 204 is `No Content`.